src/renderers/WebGLRenderer.js

```diff
@@ -2806,8 +2812,6 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	function setTexture2D( texture, slot ) {
 
-		if ( texture instanceof THREE.WebGLRenderTarget ) texture = texture.texture;
```

Comment:
the local `setTexture2D` function is only called in [this](https://github.com/mrdoob/three.js/pull/8658/files/4c4a9501a88f1e4d08fbd59306faeb18c769e890#diff-c0e88b98497597a015ecf238e91ac3a0R3052) line. The check went just above that call.